### ISSUE TITLE
सन्नन्तयङन्तधातूनां भावकर्मवाच्यरूपाणि

### ISSUE THREAD

**Issue by drdhaval2785** [drdhaval2785/SanskritVerb#661]
Page 418 S.B. part 2 is pending.
It can be taken up only after सन्नन्त and यङन्त forms are taught.
Right now pending.

**Comment by drdhaval2785** [comment]
If any other issue crops up in the process, flag a new issue.
Trial to handle is made. And sannanta and yaGanta are formed with bhAvakarma.
$vsuf (vAcya suffix) is added. 
Default is blank i.e. ''
When the vAcya is bhAva, karma or karmakartR, it becomes 'yak'.
